src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ProjectSystemOptions.cs

```diff
@@ -13,11 +12,8 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS
 {
     [Export(typeof(IProjectSystemOptions))]
-    internal class ProjectSystemOptions : IProjectSystemOptions
+    internal partial class ProjectSystemOptions : IProjectSystemOptions
```

Comment:
Why was this made `partial`?

Reply:
Ah good catch. I rejected/modified a Roslyn code fix and it was left around from that. Fixing...